### DISCUSSION TITLE
First-ever State of Green Software Report  🎉

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/discussions#10] [Announcements]
If you haven't seen it already, be sure to check out our [State of Green Software report](https://stateof.greensoftware.foundation) (SOGS). SOGS is new report that brings global insights and data from industry leaders and researchers to the forefront to reduce software's harm to Earth and increase investment in decarbonizing software at scale.

**Comment by NAMRATA-WOKE** [comment]
Interested to learn more about this publication and spread the word? Our [Marketing Toolkit](https://docs.google.com/document/d/1S7DB5_d_6qf0Bj5wzCmhhDyUXHNdaP6H9iIWaptXhis/edit?usp=sharing) has everything you need! 
Do you want to build awareness about the State of Green Software in an upcoming talk? We've got [Talking Points](https://docs.google.com/document/d/1fO29yDvRLNjyvC2f3ifmkV_RFanyNxHM8zP2oHgdu_8/edit?usp=sharing) to help you deliver key messages. 

Feel free to share the [SOGS Media Kit](https://drive.google.com/drive/folders/14x_emRJyjcON9Q2vBWIteB9b4Ij22erz?usp=sharing) for anyone interested in writing a story.
